### PR TITLE
Add Tabs-Counter

### PR DESCRIPTION
**Pull request by CoderLogy**
### Project Details
1. **What is the name of your project?**

    **Tabs-Counter**

2. **How many hours did you spend developing this?**  

    **Spent Around 2 hours on this.**

3. **What problem does your extension solve, or what makes it unique?**  

     **I was facing problems of having to know how many tabs I have open in current browser but there was no easy and intuitive way to know about it and therefore I thought to make this project because not knowing and closing important tabs made me really mad, I hope this will cure that problem. Also this is my first kind of extension I have ever made and also the knowledge I gathered from making it will help me learn JavaScript much better.**

4. **How many people have tested your extension?**  
   
    **Right now I have tested this on two different browser Edge and Firefox and hoping for others to test this.**

5. **Does your extension work in both Chrome and Firefox?**  

    **Yes it works on both the browsers.**

---

Original Github Link: https://github.com/CoderLogy/Tabs-Counter

### CODE REVIEW

**Comment by CoderLogy** [comment]
@JosiasAurel , please look into this

File: website/public/DaviLhlapakRosa_business_dino.png
Comment (JosiasAurel) [inline website/public/DaviLhlapakRosa_business_dino.png:1]: why did you delete this file? it's part of the website source code

File: submission/Tabs-Counter/chrome/popup.js
Comment (JosiasAurel) [inline submission/Tabs-Counter/chrome/popup.js:1]: How's this project unique? There are plenty other tab counter extensions out there!

**Review by JosiasAurel — CHANGES_REQUESTED** [review]
Please restore the files you deleted. Also this browser extension is not unique and I'm not sure how it's going to stop you from closing important tabs given it just shows the number of open tabs.

**Comment by CoderLogy** [comment]
What files are you asking about and why it is not unique. I had the problem of running out of memory of my computer when using browser as I couldn't stop myself from opening so many tabs. It helps right.

**Comment by JosiasAurel** [comment]
It's not clear to me how this extension prevents you from opening new tabs and running out of memory. 
If you look under the "Files Changed" tab on this pull request there are a few files that have been deleted but that shouldn't have.

File: website/public/DaviLhlapakRosa_business_dino.png
Comment (CoderLogy) [reply website/public/DaviLhlapakRosa_business_dino.png:1]: I didn't delete any file all I did was to sync it to the parent repo and any files deleted was by the parent repo not mine. I just synced the changes.

Comment (JosiasAurel) [reply website/public/DaviLhlapakRosa_business_dino.png:1]: In that case please rebase your local copy

**Comment by JosiasAurel** [comment]
Ping here. Any updates?

**Comment by CoderLogy** [comment]
Yeah just working on it, don’t have much time to do dedicate to update it.On Nov 19, 2024, at 1:01 AM, Josias ***@***.***> wrote:﻿
Ping here. Any updates?

**Comment by JosiasAurel** [comment]
> Yeah just working on it, don’t have much time to do dedicate to update it.On Nov 19, 2024, at 1:01 AM, Josias ***@***.***> wrote:﻿ Ping here. Any updates? —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

No pb. Please try to complete it. If you think you won't be able to go ahead with this, you can close the pull request.

**Comment by JosiasAurel** [comment]
Ping! Deadline is Nov 30

**Comment by JosiasAurel** [comment]
Closing due to inactivity and we're past deadline.

**Pull request closed without merging**